mypy/errors.py

```diff
@@ -597,7 +603,7 @@ def report_internal_error(err: Exception, file: Optional[str], line: int,
         for msg in errors.new_messages():
             print(msg)
     except Exception as e:
-        print("Failed to dump errors:", repr(e), file=sys.stderr)
+        print("Failed to dump errors:", repr(e), file=stderr)
```

Comment:
Note that if you change the default to `None` above, you have to be explicit here, e.g. `file=(stderr or sys.stderr)`.